.circleci/config.yml

```diff
@@ -940,10 +994,14 @@ workflows:
             - e2e-cypress
             - e2e-cypress-smoketests-local
             - e2e-cypress-public
-      - migrate:
+      - reindex:
```

Comment:
I'm honestly not sure this should be here. We really should never have to do this for prod unless maybe we're increasing the size of the ES cluster and that can't happen in-place (like we have to create the larger instance and then reindex all of the contents from the smaller instance).